fixieai/cli/agent/commands.py

```diff
@@ -149,61 +224,21 @@ def init_agent(handle, description, entry_point, more_info_url, requirement):
     current_config.more_info_url = more_info_url
     agent_config.save_config(current_config)
 
+    templator = PythonTemplator() if language == "python" else TypeScriptTemplator()
```

Comment:
probably should fail if an unknown lang is specified

Reply:
`click` handles that for us:

```
λ poetry run fixie init --language tsadfs
Usage: fixie init [OPTIONS]
Try 'fixie init --help' for help.

Error: Invalid value for '--language': 'tsadfs' is not one of 'python', 'py', 'typescript', 'TS'.
```